### DISCUSSION TITLE
Want to Generate dist with different python version.

### DISCUSSION THREAD

**Discussion by MuhammadKashif05** [dashingsoft/pyarmor#2019] [Q&A] [answered]
Hi,
I want to generate dist for my server using Python 3.10.12, while my local machine has 3.8.10.
For now I using this command to generate a dist
**pyarmor gen -O project_encrpyt/staging1 project**

when I run on server I get error

Traceback (most recent call last):
  File "/home/ubuntu/hims_pharmacy_rest_encrypt/manage.py", line 3, in <module>
    from pyarmor_runtime_006454 import __pyarmor__
  File "/home/ubuntu/hims_pharmacy_rest_encrypt/pyarmor_runtime_006454/__init__.py", line 2, in <module>
    from .pyarmor_runtime import __pyarmor__
ImportError: /home/ubuntu/hims_pharmacy_rest_encrypt/pyarmor_runtime_006454/pyarmor_runtime.so: undefined symbol: _Py_hashtable_get_entry

**Comment by jondy** [comment, marked as answer]
Please check online doc, it has been documented.
